libs/theme/src/obi.ts

```diff
@@ -1,5 +1,5 @@
-import { CustomTheme } from "./abstract";
-import { common } from "./common";
+import { CustomTheme, ImagePosition } from "./abstract";
+import { WelcomeButtons, common } from "./common";
```

Comment:
Please be consistent in our usage. `WelcomeButtons` should also be in `./abstract` and should be named `WelcomeButton`.